### PR TITLE
Dropdown fix

### PR DESCRIPTION
**Pull request by CouchofTomato**
Wrapped the dropdown event listeners in a conditional as they are not present when the user is logged out.

### CODE REVIEW

**Comment by KevinMulhern** [comment]
👍 LGTM
Thanks for this @CouchofTomato 🎉

File: app/assets/javascripts/navbar.js [hunk 1]
```diff
@@ -6,21 +6,27 @@ document.addEventListener('turbolinks:load', function() {
   var userDropdown = document.querySelector('.dropdown')
 
   // Clears Logo text in modal
-  navToggle.addEventListener('click', function() {
-    logoText.classList.toggle('expanded')
-  })
-
+  if(navToggle && logoText) {
+    navToggle.addEventListener('click', function() {
+      logoText.classList.toggle('expanded')
```
Comment (105ron) [inline app/assets/javascripts/navbar.js]: Missing semi colon

Comment (105ron) [inline app/assets/javascripts/navbar.js]: No semi-colon in this line...

File: app/assets/javascripts/navbar.js [hunk 2]
```diff
@@ -6,21 +6,27 @@ document.addEventListener('turbolinks:load', function() {
...
+  }
+  
   // Restores Logo Text on modal close
-  modalClose.addEventListener('click', function() {
-    logoText.classList.toggle('expanded');
-  })
-
+  if(modalClose) {
+    modalClose.addEventListener('click', function() {
+      logoText.classList.toggle('expanded');
```
Comment (105ron) [inline app/assets/javascripts/navbar.js]: But there's one here...

File: app/assets/javascripts/navbar.js [hunk 3]
```diff
@@ -6,21 +6,27 @@ document.addEventListener('turbolinks:load', function() {
...
   // Open User Dropdown on Hover, hide on mouseout
-  userDropdown.addEventListener('mouseover', function() {
-    userDropdown.classList.add('show')
-  })
-  userDropdown.addEventListener('mouseout', function() {
-    userDropdown.classList.remove('show')
-  })
+  if(userDropdown) {
+    userDropdown.addEventListener('mouseover', function() {
+      userDropdown.classList.add('show')
```
Comment (105ron) [inline app/assets/javascripts/navbar.js]: This is missing one too...

File: app/assets/javascripts/navbar.js [hunk 4]
```diff
@@ -6,21 +6,27 @@ document.addEventListener('turbolinks:load', function() {
...
+    })
+    userDropdown.addEventListener('mouseout', function() {
+      userDropdown.classList.remove('show')
```
Comment (105ron) [inline app/assets/javascripts/navbar.js]: Here as well...

Comment (CouchofTomato) [inline app/assets/javascripts/navbar.js]: I left all other code how it was and just added the conditional

**Pull request merged**